src/database/hooks/useSaveAnonChatHook.ts

```diff
@@ -1,3 +1,4 @@
+import React from 'react';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `React` is added but it seems unnecessary as there are no JSX elements in this file. If you're using React hooks, they can be imported directly without importing the entire `React` object.
```diff
- import React from 'react';
+ import { useState } from 'react';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->